server/subject_transform.go

```diff
@@ -0,0 +1,506 @@
+// Copyright 2018-2023 The NATS Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"fmt"
+	"hash/fnv"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+var commaSeparatorRegEx = regexp.MustCompile(`,\s*`)
```

Comment:
These in fast path?

Reply:
Nope, regex are compiled when the server starts and then only used when you create the transform, not when the transform is actually used.